builder/requirements.txt

```diff
@@ -1 +1 @@
-pymongo
```

Comment:
This was the only entry. The file is empty now - should I just remove it?

Reply:
Yes, but we will also need some cleanup in `builder/Dockerfile` (which references this file, should just be a single `RUN` entry to remove).